dotcom-rendering/src/web/components/Footer.tsx

```diff
@@ -143,30 +146,65 @@ const footerList = css`
 const readerRevenueLinks = css`
 	border-left: ${footerBorders};
 	flex: 1;
-	padding: 12px 0 0 10px;
-	margin: 0 10px 36px 0;
+	padding: ${space[3]}px 0 0 10px;
+	margin: 0 10px ${space[9]}px 0;
 	width: calc(50% - 10px);
+
 	${until.tablet} {
 		width: 50%;
 		border-top: ${footerBorders};
 	}
 `;
 
+const acknowledgments = css`
+	grid-area: acknowledgment;
+	align-self: end;
+	${textSans.xxsmall()};
+	padding-top: ${space[3]}px;
+	padding-bottom: ${space[3]}px;
+	margin-bottom: ${space[6]}px;
+
+	${between.tablet.and.desktop} {
+		padding-right: ${space[3]}px;
+		max-width: 481px;
+		border-right: ${footerBorders};
+		position: relative;
+
+		&::after {
+			content: '';
+			position: absolute;
+			height: ${space[9]}px;
+			top: -${space[9]}px;
+			right: -1px;
+			border-right: ${footerBorders};
+		}
+	}
```

Comment:
This is [as per the design](https://www.figma.com/file/eT6V6c69KxPhKmLReFfZ08/Australia-Footer?node-id=1%3A2). The vertical divider needs to span accross the empty space.